### ISSUE TITLE
Hae a different attraction check if she peed in a strange place before

### ISSUE THREAD

**Issue by FamTroder** [FamTroder/FamMel#27]
For example in  peevase attraction needs to be 30 for her not to immediately deny the idea. But maybe make it like 15 if she has peed in the vase before?

**Comment by M0nkey5ee** [comment]
Good idea, maybe this should reflect in the dialogue as well, not just in the code?

**Comment by Forget-About-Me2** [comment]
Yeah, think that is an implicit requirement of this, indeed. Make her mention it or something
